main.py

```diff
@@ -95,7 +96,10 @@ def change_read():
         book.save()
     else:
         print('\nThe book ID you have entered is NOT in the database\n')
-    
+
+def number_of_books():
+    books = store.get_all_books() # gets all the books in the database
```

Comment:
this is more work on the db. there is a book_count function in the bookstore class that you could have used. read code and know what it already does before you modify it. https://github.com/4LineTime/readinglist/blob/bbff186a5e09b6c91eb5f795771650486327cef9/bookstore.py#L255-L266